configure

```diff
@@ -3288,7 +3300,7 @@ if ${nl_cv_protoc_version+:} false; then :
   $as_echo_n "(cached) " >&6
 else
 
-		       nl_cv_protoc_version=`${PROTOC} --version 2>&1 | ${SED} -n -e 's/^.\{1,\}\([0-9]\{1,\}.[0-9]\{1,\}.[0-9]\{1,\}\)$/\1/gp'`
+		       nl_cv_protoc_version=`${PROTOC} --version 2>&1 | ${SED} -n -E -e 's/^[^0-9]*(([0-9]+\.)*[0-9]+).*/\1/p'`
```

Comment:
@gerickson you cool with this regex change?

Reply:
This should be auto-generated and should match what came in via Eli Ribble from commit ea8339e

Reply:
Note: this regex change has been previously approved and pulled into `configure.ac` in https://github.com/openweave/openweave-wdlc/pull/39 ; in this pull request we simply update the automake-generated files. The change was made to support protoc-3.6.1 and similar style of versions